### PR TITLE
Import workshop

### PR DESCRIPTION
**Pull request by jburel**
Work in progress

### CODE REVIEW

File: 2018/Users-Meeting/Workshops/Metadata-Import/slides/index.html [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+            <!-- Any section element inside of this container is displayed as a slide -->
+            <div class="slides">
+
+                <!-- title page -->
+                <section>
+                    <h1>The journey of Data</h1>
+                    <p>
+                        OME Users meeting<br>
+                        31th May 2018<br>
```
Comment (mtbc) [inline 2018/Users-Meeting/Workshops/Metadata-Import/slides/index.html]: 31st

**Comment by pwalczysko** [comment]
Why is on slide named

```
Post Import Data Manipulation: Demo.
```

the order first ``What to render..``, second ``How ot render...`` ? On the previous slide (bulk import) the order is exactly the oppsite (first Hwo, then What). I would suggest stick to the order 

1. How
2. What

File: 2018/Users-Meeting/Workshops/Metadata-Import/slides/index.html [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+                    <ul>
+                        <li>What to render: <i>renderingMapping.tsv</li>
+                        <pre style="font-size:40%; overflow: scroll"><code>
+CDK5RAP2-C_     renderingdef.yml
+CENT2_  renderingdef2.yml
+CEP120/20111106_        renderingdef.yml
+CEP120/20111209_        renderingdef2.yml
+...
+                        </code></pre>
+                        <li>How to render: <i>renderingdef.yml</li> & <i>renderingdef2.yml</li>
```
Comment (pwalczysko) [inline 2018/Users-Meeting/Workshops/Metadata-Import/slides/index.html]: This whole section renders as italics. I think it should be normal, not italics, except the filenames.

**Comment by pwalczysko** [comment]
Maybe less crucial, but the two lines below are the most important on the whole graphics. Would it be possible to make them more prominient than grey ? (the semi-dotted line is even worse to see than the ``Create links`` one)

screen shot 2018-05-25 at 20 32 46

**Comment by pwalczysko** [comment]
3 comments, otherwise looks nice.

**Comment by jburel** [comment]
I will need to dig for the original file. Not sure I will be able to find it

**Comment by joshmoore** [comment]
@mtbc may know where the http://ditaa.sourceforge.net/ (?) source files are.

**Comment by mtbc** [comment]
That doesn't look like a `ditaa` image to me. My source files are the `*.ad` files in https://github.com/ome/presentations/tree/master/2013/fs-workshop-paris/diagrams.

**Comment by mtbc** [comment]
Don't be distracted by `arrow.ps`, that's a curvy red one. Perhaps for some other diagram the PNGs from ditaa were fiddled with by something else.

**Comment by sbesson** [comment]
Anything coming here @jburel. No objection to adding the import PNG to the overall templates. If there are concerns, we could move them back to a specific `images` folder

**Comment by jburel** [comment]
Discussed with @mtbc, he thinks that the png was edited after the fact via maybe photoshop for the dotted line
so due to the time constraint I will keep the existing png so nothing more to come

**Pull request merged**